### ISSUE TITLE
Unable to train the agent

### ISSUE THREAD

**Issue by puneets2811** [Unity-Technologies/ml-agents#522]
Followed the steps from "https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Learning-Environment-Create-New.md" to create a car with 5 radars(as shown in the image) on the front bumper and controls - break, steer, accelerate.
![untitled_1](https://user-images.githubusercontent.com/18008801/37697407-b63ddd24-2d02-11e8-8722-cca307c7e9d8.png)

When I execute the "learn.py" script with the Unity executable of my environment in the same directory, the unity environment opens up but my agent does not move or learn anything and no errors appear. On the python console a constant reward of -1.000 is shown over the iterations.

**Comment by vincentpierre** [comment]
Do you obtain the expected behavior when using the player brain ?

**Comment by puneets2811** [comment]
Yes. If the player brain is set, and I move the car manually using the keyboard keys, the reward goes up.

**Comment by Bakikii** [comment]
Have you ensured rewards are present to encourage changes in behaviour....

**Comment by BenM77** [comment]
I have a similar problem. My environment runs and the player brain type works perfectly, but the agent does not move at all during training. A couple errors happen repeatedly:
